one-pager.md

```diff
@@ -1,33 +1,47 @@
+* Room for creative projects and enterprises
+* Public access spaces and events
 
-**Community**: We believe that resident wellbeing and community wellbeing is interdependent.
-* Shared common house with multiple dwelling units for privacy
-* Stability of homeownership with the flexible commitment of renting
-* Public communal spaces including public orchards and gardens with [Help Yourself](http://commongreen.weebly.com/)
-* Supportive of local enterprises and creative projects
-* Collective decision making and conflict mediation with [Sociocracy For All](https://www.sociocracyforall.org/)
+**Resident Wellbeing**
+* Flexible commitment of renting
+* Security and control of traditional homeownership
+* Community of cohousing or ecovillage without financial inaccessibility or cultural conformity/ too intentional? 
 
-**Equity**: We believe housing is a right and should be affordable and democratically governed.
+**Democratic** 
 * Housing co-op will own assets as a group, with residents as equal voting members
+* Collective decision making and conflict mediation
+
+**Affordable** 
 * [Group equity](https://www.nasco.coop/development/handbook/equity) model will ensure low move in share, to be returned when they move out
 * Initial funds will be raised from grants and community investors, not residents
-* Sliding scale monthly costs will be reinvested in the property and used to pay off investors
-* We are working with [Cooperative Development Institute](https://cdi.coop/) to legally incorporate
+* Residents will pay monthly costs to repay community investors
+
+We aim to live sustainably through community stewardship for a [just transition](https://movementgeneration.org/wp-content/uploads/2016/11/JT_booklet_English_SPREADs_web.pdf 
 
-**Sustainability**: We believe in closed-loop, regenerative systems to support abundance and resilience as part of a [just transition away](https://movementgeneration.org/wp-content/uploads/2016/11/JT_booklet_English_SPREADs_web.pdf) from an extractive economy.
-* Net-zero buildings with help from [Upper Valley Cottage and Cabin](https://www.instagram.com/uppervalleycc/) and [Vermod](https://vermodhomes.com/)
-* Member of [Co-op Power](https://www.cooppower.coop/rays-the-valley), which develops community solar and creates local green jobs
-* Food sovereignty through regenerative agriculture 
+**Just**
+* Housing and land should be decommodified and stewarded by and for the community, not speculative profits.
+* Closed-loop, regenerative systems to support abundance and resilience away from an extractive economy.
 
-**Stewardship**: We believe land should be stewarded for the community, not speculative profits.
+**Sustainability**:
+* Net-zero buildings using passive solar 
+* Member of renewable energy co-op
+* Food sovreignty through regenerative agriculture 
+
+**Stewardship**: 
 * A community land trust will own the land permanently, and have a say in resale
```

Comment:
```suggestion
* The community with have a permenant say in the land, through a community land trust owning it
```

Something like that? Or at least start with what the end result is, then get into particulars?